ui/app/adt/controllers/adtController.js

```diff
@@ -29,7 +29,23 @@ angular.module('bahmni.adt')
                 }
                 return defaultVisitTypeUuid;
             };
+            $scope.translateDispositionForBedManagement = function (attribute, moduleName) {
+                if (typeof attribute != 'undefined') {
+                    if ((moduleName == null) || (typeof moduleName == 'undefined')) {
+                        var keyPrefix = " ";
+                    } else {
+                        keyPrefix = moduleName;
+                    }
 
+                    var keyName = attribute.toUpperCase().replace(/\s\s+/g, ' ').replace(/[^a-zA-Z0-9 _]/g, "").trim().replace(/ /g, "_");
+                    var translationKey = keyPrefix + "_" + keyName;
+                    var translation = $translate.instant(translationKey);
+                    if (translation != translationKey) {
+                        attribute = translation;
+                    }
+                }
+                return attribute;
+            };
```

Comment:
@buvaneswari-arun we already have this method in the translationUtil.js. Then why are we not using it here ?

Reply:
TranslationUtils.translateAttribute method is not accessible from this bedmanagement. Hence forced to put the content of the method directly here.

Reply:
@buvaneswari-arun is it the same case with other modules ?

Reply:
I tried with Bedmanagement - as it was part of the. card - not sure about IPD and programs etc and they will be played in the upcoming releases